Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -916,7 +920,9 @@ extension BrowserViewController: HomePanelViewControllerDelegate {
         finishEditingAndSubmit(url, visitType: visitType)
     }
     func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectPanel panel: Int) {
-        tabManager.selectedTab?.lastHomePanel = panel
+        if AboutUtils.isAboutHomeURL(tabManager.selectedTab!.url!) {
```

Comment:
Same here re: `nil` URLs. To make life easier, you could make `isAboutHomeURL` accept an `NSURL?` instead of an `NSURL` -- then we wouldn't need to unwrap it.